src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -300,6 +300,19 @@ public static boolean isDaemonInitialized()
         return daemonInitialized;
     }
 
+    /**
+     * Enables memtable initialization via {@link org.apache.cassandra.db.ColumnFamilyStore}.
+     */
+    public static void setEnableMemtableInitialization()
+    {
+        enableMemtableInitialization = true;
+    }
+    
+    public static boolean enableMemtableInitialization()
```

Comment:
I think we should port `enableMemtableAndCommitLog()` from DSE entirely. There are a few usages which are important for multi-region repair service. eg. `Tracker#isDummy` and `CompactionManager#doValidationCompaction`

Reply:
Ok. I misunderstood then, thinking we only wanted the memtable init piece.